application/apps/indexer/session/Cargo.toml

```diff
@@ -6,18 +6,22 @@ edition = "2021"
 
 [dependencies]
 crossbeam-channel = "0.5"
+dirs = "4.0"
 futures = "0.3"
 indexer_base = { path = "../indexer_base" }
 log = "0.4"
 merging = { path = "../merging" }
 parsers = { path = "../parsers" }
 processor = { path = "../processor" }
+rustc-hash = "1.1"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 sources = { path = "../sources" }
 thiserror = "1.0"
 tokio = { version = "1.24", features = ["full"] }
 tokio-stream = "0.1"
 tokio-util = "0.7"
-uuid = { version = "1.2", features = ["serde", "v4"] }
-dirs = "4.0"
\ No newline at end of file
+uuid = { version = "1.0", features = ["serde", "v4"] }
```

Comment:
why go back to an old version? current version should be `"1.3"`

Reply:
Corrected to 1.2. as most recent (1.2.2)